manifests/init.pp

```diff
@@ -176,7 +176,7 @@
       owner  => 'root',
       group  => 'root',
       mode   => '0644',
-      source => "puppet:///modules/epel/RPM-GPG-KEY-EPEL-${os_maj_release}",
+      content => template("epel/RPM-GPG-KEY-EPEL-${os_maj_release}.erb"),
```

Comment:
I actually thought everyone was already using ERB and EPP templates in this way for the same reasons. Certainly I have seen other module authors migrate their files into templates for the same reason. Not only does it work on masterless Puppet, it means we remove a dependency on an external system; it avoids the quirky behaviour of Puppet adding everything into the catalog- except file content; and it means users only need to know one way of populating files with content. Happy to add a comment though if you think it helps. Do you want the comment in the manifest, in the tests, or both?